src/main/java/org/example/Medication.java

```diff
@@ -4,6 +4,27 @@ public class Medication {
   private String brand_name;
 
   private String purpose;
+
+  private String strength;
+
+  private String dosage_form;
+
+    public String getStrength() {
+        return strength;
+    }
+
+    public void setStrength(String strength) {
+        this.strength = strength;
+    }
+
+    public String getDosage_form() {
+        return dosage_form;
+    }
+
+    public void setDosage_form(String dosage_form) {
+        this.dosage_form = dosage_form;
+    }
+
     public String getBrand_name() {
         return brand_name;
     }
```

Comment:
Have you thought about using the Lombok library for getters and setters?  It provides annotations that generates getter and setter methods automatically at compile time. It can reduce the code in this class if you decide to add more fields for this object.

Reply:
yeah i know lombok. I'm using Automatter in the other repo and it does the same thing. I guess since i was using this repo as a playground i didn't really care about the code? but it wouldn't hurt to get some more practice in!  Next version will have one of the two